.pre-commit-config.yaml

```
@@ -31,12 +31,11 @@ repos:
       - id: check-toml
       - id: check-yaml
       - id: check-ast
--   repo: https://github.com/python-poetry/poetry
-    rev: "2.1.3"
+-   repo: https://github.com/astral-sh/uv-pre-commit
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)